### ISSUE TITLE
RDP Only Option

### ISSUE THREAD

**Issue by nabber00** [LastSquirrelIT/MultiOneTimePassword-CredentialProvider#13]
Would be nice to have an RDP only option.  I don't know how that is implemented but the Duo Security Windows Logon Credential Provider can do it with a registry setting.  It might be checking for a remote IP address.

**Comment by arcadejust** [comment]
I have the "RDP only" switch in my version but I don't know how to upload it. I have made my version for the Credential V2 (windows 8 and 10 style) so it's not a branch but rather completely different project... I kept the original logo:D and I can verify that the microsoft account 2'nd level authentication works with multiotp !!! If you like I can send you the code and dll

**Comment by nabber00** [comment]
Yeah that sounds perfect.  I'm only using Windows 8.1+ these days anyway.  Also great info on how to check for RDP, should be easy to patch that in.

**Comment by arcadejust** [comment]
I have just uploaded my first project with a git repository... I don't know if it's any good, but hey I've never used git before :D you could try this https://github.com/arcadejust/MultiotpCPV2RDP

**Comment by nabber00** [comment]
Looks very simple, which I like. If you want to tag it as a release you can upload any binaries that go with it into the "releases" tab.

**Comment by arcadejust** [comment]
It's ready - you can try it out

**Comment by nabber00** [comment]
What do I do with the DLL?  Does it go in a particular path?  Do I need to register it?

**Comment by arcadejust** [comment]
First make sure you have multiotp installed and the path to it is correct in register.reg (default "MultiOTPPath"="c:\\multiotp\\windows\\" modify the path if you have to with notepad).Copy the dll to c:\windows\system32 and then run register.reg. Right now the setting "MultiOTPRDPOnly"=dword:00000001 suppress CP for local login and lock screen so you will not see a credential provider from your console, you will have to open rdp from another computer to see it.

**Comment by nabber00** [comment]
Yeah that worked as expected, great!
